exercises/concept/the-farm/.meta/design.md

```diff
@@ -0,0 +1,7 @@
+# Design
+
+## Goal
+
+- Know how to return an error
+- Know how to check an error
+- Know when and how to return as soon as an error occurs.
```

Comment:
I'd probably add:

```suggestion
- Know how to create an error
- Know how to return an error
- Know how to check an error
- Know when and how to return as soon as an error occurs.
```

Reply:
You might also consider:
- teaching wrapping of errors: `fmt.Errorf("Some error: %w", err)`
- teaching checking for error types: `errors.Is(err, SomeErrType)`